main.js

```diff
@@ -24,11 +24,12 @@ function run(cmd, options = {}) {
     });
 }
 
-run(`$(aws ecr get-login --no-include-email --region ${awsRegion})`);
-
+const accountLoginPassword = run(`aws ecr get-login-password`);
 const accountData = run(`aws sts get-caller-identity --output json`);
 const awsAccountId = JSON.parse(accountData).Account;
 
+run(`$(docker login --username AWS ${accountLoginPassword} ${awsAccountId}.dkr.ecr.${awsRegion}.amazonaws.com)`);
```

Comment:
you missed `-p`  

this line should be 

```
run(`$(docker login --username AWS -p ${accountLoginPassword} ${awsAccountId}.dkr.ecr.${awsRegion}.amazonaws.com)`);
```

Reply:
now this line is when the `run` runs its running in two lines like 

```
docker login --username AWS -p ${accountLoginPassword} 
${awsAccountId}.dkr.ecr.${awsRegion}.amazonaws.com
```

Making impossible login on ecr repo

Reply:
PR #11  fix it